shared/src/androidMain/kotlin/dev/johnoreilly/confetti/work/SessionNotificationSender.kt

```diff
@@ -70,13 +68,19 @@ class SessionNotificationSender(
         }
 
         val intervalRange = createIntervalRange()
-        val sessionsToNotify = bookmarkedSessions.filter { session ->
+        var sessionsToNotify = bookmarkedSessions.filter { session ->
```

Comment:
why var?

Reply:
For absolutely no reason. 🙃  Probably a left-over from my experiments. Will remove it in a follow-up PR.